packages/cli/src/__tests__/main.test.ts

```diff
@@ -100,9 +106,20 @@ describe('main', () => {
         name: 'my-test-plugin',
         output: './dist/index.js',
         dir: 'my-directory',
+        publish: true,
         dotEnvPath: '.',
         scope: 'my-plugins',
       })
     })
+
+    it('--no-publish', () => {
+      const cli = createCLI()
+      cli.parse([...DEFAULT_ARGS, 'deploy', ' --no-publish'])
```

Comment:
```suggestion
      cli.parse([...DEFAULT_ARGS, 'deploy', '--no-publish'])
```
Maybe the space is affecting the test case?

Reply:
yes indeed, just fixed it thank you 🙏